src/CLPUtils.h

```diff
@@ -29,56 +29,59 @@
 #define CLP_XSTRCAT3(a, b, c) CLP_STRCAT3(a, b, c)
 
 #if defined(CLP_DEBUG_PORT)
-extern int CLP_printf(const char* format, ...);
-extern int CLP_printf(const __FlashStringHelper* format, ...);
-#define clp_printf CLP_printf
+namespace CLP {
+extern int printf(const char* format, ...);
+extern int printf(const __FlashStringHelper* format, ...);
```

Comment:
what das the `...` mean here, I don't know this syntax

Reply:
It's a special symbol for [variadic functions](https://en.cppreference.com/w/cpp/utility/variadic). It allows a variable amount of arguments which is useful for formatting with printf.

Reply:
Ok I didn't know you can put a comma there